linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/ConfigurationRestfulApi.java

```diff
@@ -488,7 +488,7 @@ public Message saveKeyValue(HttpServletRequest req, @RequestBody Map<String, Obj
     try {
       configurationService.paramCheck(configKeyValue);
     } catch (Exception e) {
-      if (force) {
```

Comment:
catch   ConfigurationException

Reply:
The method did not throw this exception, so it cannot catch~